### PR TITLE
Add dev.ares.ares

### PR DESCRIPTION
**Pull request by ghisvail**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*

Requires `--device=all` for controller access.

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build dev.ares.ares

**Comment by LeandroStanger** [comment]
bot, build dev.ares.ares

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

File: dev.ares.ares.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+app-id: dev.ares.ares
+runtime: org.gnome.Platform
```
Comment (hfiguiere) [inline dev.ares.ares.yaml:2]: do you really need the  GNOME platform? Isn't the freedesktop SDK enough?

Comment (ghisvail) [reply dev.ares.ares.yaml:2]: I probably could, but I get issues with icons which I am not getting with the GNOME runtime.

Comment (ghisvail) [reply dev.ares.ares.yaml:2]: `Error loading theme icon 'document-open' for stock: Icon 'document-open' not present in theme Adwaita`

Comment (ghisvail) [reply dev.ares.ares.yaml:2]: I am not getting this issue with `bsnes` though which is weird considering both code bases are pretty similar.

Comment (hfiguiere) [reply dev.ares.ares.yaml:2]: fair enough.

(not sure how all of this work)

Comment (ghisvail) [reply dev.ares.ares.yaml:2]: I'll ask on our matrix channel just in case

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

**Comment by barthalion** [comment]
Anything left to do here? It's marked as WIP but looks good to me.

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

**Comment by ghisvail** [comment]
bot, build dev.ares.ares

**Comment by ghisvail** [comment]
> Anything left to do here? It's marked as WIP but looks good to me.

I think it's ok now :+1:

**Pull request closed without merging**